components/tdisplays3/display.py

```diff
@@ -112,6 +119,13 @@ async def to_code(config):
     await display.register_display(var, config)
     cg.add(var.set_dimensions(config[CONF_WIDTH], config[CONF_HEIGHT]));
 
+    if CONF_USE_ASYNC_IO in config:
+        if config[CONF_USE_ASYNC_IO]:
```

Comment:
This could be shortened: 
```suggestion
    if config.get(CONF_USE_ASYNC_IO, False):
```

Reply:
And actually, since a default is set, it shouldn't even be needed; you could just check `if config[CONF_USE_ASYNC_IO]:` (but it's safer to check both, of course)

Reply:
Totally agree